extensions/token-services/token-services-spi/src/main/java/org/eclipse/dataspaceconnector/token/spi/ValidationRule.java

```diff
@@ -0,0 +1,11 @@
+package org.eclipse.dataspaceconnector.token.spi;
+
+import com.nimbusds.jwt.JWTClaimsSet;
+import org.eclipse.dataspaceconnector.spi.result.Result;
+import org.jetbrains.annotations.NotNull;
+
+@FunctionalInterface
+public interface ValidationRule {
```

Comment:
Couldn't be merged with the other `ValidationRule` interface and put on a upper level like a `token-spi`? They offer pretty much the same functionalities

Reply:
Yeap that s something I have in mind, to have some common stuff to manage and validate tokens. But for now this would lead to introduce dependency on nimbus library in spi module, which we probably don't want to do. 
This requires further discussion, so we propose to keep it like this for now.